src/stories/Forms/PhoneInput/index.tsx

```diff
@@ -0,0 +1,157 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import {
+  parsePhoneNumber,
+  getSupportedCallingCodes,
+  getRegionCodeForCountryCode,
+  getCountryCodeForRegionCode,
+  getAsYouType,
+  getExample,
+} from 'awesome-phonenumber';
+import { GlobalInputProps } from '../../../shared/types';
+import { Select, Input, Flex } from '../../../';
+
+interface WrapperProps {
+  hasError?: boolean;
+}
+
+const Wrapper = styled(Flex)<WrapperProps>`
```

Comment:
Sometimes when it is just one prop like this I can think it is easier to read to just have it in curly brackets inside. So styled(Flex)<{hasError?: booleam}>. But no need to change this just a random thought while reading:)